10/centos-7/rootfs/libpostgresql.sh

```
@@ -481,7 +496,7 @@ postgresql_initialize() {
             rm "$postmaster_path"
         fi
         is_boolean_yes "$create_pghba_file" && postgresql_restrict_pghba
-        is_boolean_yes "$create_conf_file" && postgresql_configure_replication_parameters
+        is_boolean_yes "$create_conf_file" && postgresql_configure_replication_parameters && postgresql_configure_fsync
```

To avoid masking any possible error that could occur in the function `postgresql_configure_replication_parameters`, I'd use:
```diff
-         is_boolean_yes "$create_conf_file" && postgresql_configure_replication_parameters && postgresql_configure_fsync
+         is_boolean_yes "$create_conf_file" && postgresql_configure_replication_parameters
+         is_boolean_yes "$create_conf_file" && postgresql_configure_fsync

```